src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObj = sourceString
+    .split(';')
+    .map((itemFromArr) => itemFromArr.trim())
+    .filter((itemFromArr) => itemFromArr.length > 0)
+    .map((itemFromArr) => {
+      const separatorIndex = itemFromArr.indexOf(':');
+      const newKey = itemFromArr.slice(0, separatorIndex).trim();
+      const newValue = itemFromArr.slice(separatorIndex + 1).trim();
+
+      return [newKey, newValue];
+    })
+    .reduce((acc, [key, value]) => {
+      acc[key] = value;
+
+      return acc;
+    }, {});
+
+  return resultObj;
```

Comment:
The variable name 'resultObj' is not descriptive of its contents. Consider renaming it to something more meaningful, such as 'styleObject' to improve code readability.